### PR TITLE
[WOE] Implement The Goose Mother

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[The Goose Mother]]

File: Mage/src/main/java/mage/abilities/dynamicvalue/HalfValue.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    @Override
+    public int calculate(Game game, Ability sourceAbility, Effect effect) {
+        int base = value.calculate(game, sourceAbility, effect);
+        if (base % 2 == 0) {
+            return base / 2;
+        } else {
+            return 1 + base / 2;
+        }
```
Comment (theelk801) [inline Mage/src/main/java/mage/abilities/dynamicvalue/HalfValue.java:40]: this doesn't actually utilize `roundedUp` when calculating the value

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage/src/main/java/mage/abilities/dynamicvalue/HalfValue.java (hunk 1 above)
Comment (Susucre) [reply Mage/src/main/java/mage/abilities/dynamicvalue/HalfValue.java:40]: Oh right, thanks for noticing.
That would have been an interesting bug if that ever would have been used with round down.

Comment (Susucre) [reply Mage/src/main/java/mage/abilities/dynamicvalue/HalfValue.java:40]: And I just found out we actually had an HalfValue class already.

**Pull request merged**